PWGMM/Mult/Tasks/dndeta-mft.cxx

```diff
@@ -35,20 +35,35 @@
 using namespace o2;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
+using namespace o2::aod::track;
```

Comment:
Why is this needed? Further in the code you still use `aod::track::` namespace prefix.

Reply:
This is needed for all the definitions of the TrackSelectionFlags, I think it will be a bit too long if I remove it and write aod::track:: everywhere